datastore/client.go

```diff
@@ -42,11 +42,15 @@ type datastoreClient struct {
 	md metadata.MD
 }
 
-func newDatastoreClient(conn grpc.ClientConnInterface, projectID string) pb.DatastoreClient {
+func newDatastoreClient(conn grpc.ClientConnInterface, projectID, databaseID string) pb.DatastoreClient {
+	resourcePrefixHeaderVal := "projects/" + projectID
```

Comment:
nit: this could be just `v` or `resourcePrefixValue` to match the key if you prefer a long name.

Reply:
Done